routes/auth.js

```diff
@@ -0,0 +1,27 @@
+const express = require("express");  // pentru a crea servere web și a gestiona rutele API.
+const router = express.Router();     // submodul ce ajută la organizarea și modularizarea rutelor aplicației
+const dotenv = require('dotenv');    // pentru  încărcarea variabilelor de mediu dintr-un fișier .env în aplicație
+const jwt = require('jsonwebtoken'); // pentru crearea și validarea tokenurilor JWT
+const sha256 = require('sha256');    // pentru criptarea și compararea parolelor
+const DB_URL = process.env.DB_URL;
+
+dotenv.config();
+
+router.post('/autentificare-admin', async (req, res) => {
```

Comment:
Denumirea rutelor trebuie sa fie inrromana, ca si denumirile variabilelor

comentariile pot sa fie si in romana, dar sa te obistunesti sa le pui doar acolo unde este necesar, unde nu se intelege la prima vedere ce se intampla